### DISCUSSION TITLE
Split ssh session with specific cwd instead of current

### DISCUSSION THREAD

**Discussion by crater2150** [kovidgoyal/kitty#8134] [Q&A]
I know that passing `--cwd=current` to the `launch` command in a ssh session will open the split also on the remote server, this works fine. But passing `--cwd=/some/other/path/on/remote`  will open a local split, trying to open the remote path locally. I'm using this in a script that starts several programs in splits, which need different working directories.

Is there some other parameter that forces `launch` to open the new window on the same host as the current one? I've tried `--copy-env`, but that does not seem to cause an ssh connection either.

As a workaround, I now launch a wrapper script that changes directory and then executes the command, but it would be nice if there was a way without an extra script.

**Comment by kovidgoyal** [comment]
Does

launch kitten ssh --kitten cwd=/path/on/remote servername

not work?

**Reply by crater2150** [reply]
That does work, but I also use the same script locally sometimes, so it would need to check where it is running to decide wether to run the commands with the ssh kitten. To clarify, currently I connect via ssh and then launch the script _on the remote host_, or I run it locally on the same host (which is a desktop that I sometimes access remotely).

I just hoped there was a variant that worked the same with or without ssh (like when passing `--cwd=current`), something like a `--copy-host` parameter.
